bin/dsmd.pl

```diff
@@ -532,6 +532,7 @@ sub manage_alarms {
 
 sub run {
     my ($self, %options) = @_;
+    my $dbCentreon = 'centreon';
```

Comment:
create a variable does not fix anything, you should get this value from config, like it's done for centstorage connection

Reply:
something like 
```
    $self->{db_centreon} = centreon::common::db->new(
        db => $self->{centreon_config}->{centreon_db},
        host => $self->{centreon_config}->{db_host},
        port => $self->{centreon_config}->{db_port},
        user => $self->{centreon_config}->{db_user},
        password => $self->{centreon_config}->{db_passwd},
        force => 0,
        logger => $self->{logger}
    );
    $self->{db_centreon}->connect();
```

Reply:
done